kythe/go/extractors/config/runextractor/backup/backup.go

```diff
@@ -40,7 +40,7 @@ import (
 // file.  The Restore method replaces the contents of the original with the
 // backup, overwriting any changes that were made since the backup was created.
 type File struct {
-	orig, tmp string // file paths
+	Orig, Tmp string // file paths
```

Comment:
it doesn't look like these need to be exported?

Reply:
Sorry, I missed that this was used for printing the differences.  Perhaps unexport these but include a function that reports the differences directly?

I think you might also want to use https://godoc.org/github.com/google/go-cmp/cmp rather than shelling out to the `diff` command.

Reply:
Done.